openpype/hosts/maya/api/lib.py

```diff
@@ -196,36 +201,48 @@ def panel_camera(panel, camera):
         cmds.modelPanel(panel, edit=True, camera=original_camera)
 
 
-def playblast_capture(preset, instance):
-    """Function for playblast capturing with the preset options
+def render_capture_preset(preset):
+    """Capture playblast with a preset.
+
+    To generate the preset use `generate_capture_preset`.
 
     Args:
         preset (dict): preset options
-        instance (str): instance
 
     Returns:
-        _type_: _description_
+        str: Output path of `capture.capture`
     """
+
+    # Force a refresh at the start of the timeline
+    # TODO (Question): Why do we need to do this? What bug does it solve?
+    #   Is this for simulations?
+    cmds.refresh(force=True)
+    refresh_frame_int = int(cmds.playbackOptions(query=True, minTime=True))
+    cmds.currentTime(refresh_frame_int - 1, edit=True)
+    cmds.currentTime(refresh_frame_int, edit=True)
+
     if os.environ.get("OPENPYPE_DEBUG") == "1":
         log.debug(
             "Using preset: {}".format(
                 json.dumps(preset, indent=4, sort_keys=True)
             )
         )
+
+    # not supported by `capture` so we pop it off of the preset
+    reload_textures = preset["viewport_options"].pop("reloadTextures", True)
+
     with ExitStack() as stack:
         stack.enter_context(maintained_time())
-        stack.enter_context(panel_camera(
-            instance.data["panel"], preset["camera"]))
-        stack.enter_context(viewport_default_options(preset, instance))
+        stack.enter_context(panel_camera(preset["panel"], preset["camera"]))
+        stack.enter_context(viewport_default_options(preset))
         if preset["viewport_options"].get("textures"):
-            stack.enter_context(material_loading_mode())
-            if preset["viewport_options"].get("reloadTextures"):
+            # Force immediate texture loading when to ensure
+            # all textures have loaded before the playblast starts
+            stack.enter_context(material_loading_mode("immediate"))
+            if reload_textures:
                 # Regenerate all UDIM tiles previews
                 reload_all_udim_tile_previews()
-        # not supported by `capture`
-        preset["viewport_options"].pop("reloadTextures", None)
         path = capture.capture(log=self.log, **preset)
```

Comment:
```suggestion
        preset.pop("panel")
        path = capture.capture(log=self.log, **preset)
```
do we need to pop `preset["panel"]` before the capture, as the argument from the capture function doesn't include `panel`?

Reply:
If it doesn't have the argument, then yes. I thought it had.